core/scripts/go.mod

```diff
@@ -363,5 +363,10 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-// replicating the replace directive on cosmos SDK
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+replace (
+	// until merged upstream: https://github.com/omissis/go-jsonschema/pull/264
+	github.com/atombender/go-jsonschema => github.com/nolag/go-jsonschema v0.16.0-rtinianov
```

Comment:
Adding this `replace` directive to match common: https://github.com/smartcontractkit/chainlink-common/blob/main/go.mod#L111